openassessment/xblock/static/js/src/lms/oa_base.js

```diff
@@ -68,6 +69,108 @@ OpenAssessment.BaseView.prototype = {
     },
 
     /**
+     * Check if the Aria live region already exists, if it doesn't then add it.
```

Comment:
Nit: I'd reword this (the important part of the method is the reading of the text messages)--

Add the text messages to the Aria live region (first creating it if necessary).